packages/classic/README.md

```diff
@@ -2,8 +2,119 @@
 
 The Kendo UI Classic Theme is a SCSS-based theme of the Less based themes for the Kendo UI components.
 
+* [Quick start](#quick-start)
+* [Customizing](#customizing)
+* [Integrating with third party frameworks](#integrating-with-third-party-frameworks)
+* [Bug reports and feature requests](#bug-reports-and-feature-requests)
+* [Versioning](#versioning)
 * [License](#license)
 
+## Quick start
+
+Kendo UI Classic Theme is available as an npm package.
+
+If you don't need to apply any [customization](#customizing) to the theme, you can can reference the precompiled CSS directly from unpkg CDN:
+
+```html
+<link rel="stylesheet" href="https://unpkg.com/@progress/kendo-theme-classic/dist/all.css" />
+```
+
+While using the precompiled CSS file is faster than compiling the theme from the source code, the approach has the two drawbacks: it includes CSS for all components; it does not provide options for theme customization through SCSS variables (which is possible when you build the theme from the source code) because the theme is already compiled.
+
+If you prefer, or you need to customize the theme, you can install it:
+
+```sh
+npm install --save @progress/kendo-theme-classic
+```
+
+and then import it in your project styles:
+
+```scss
+// Import the entire theme
+@import "~@progress/kendo-theme-classic/dist/all.scss";
+```
+
+For more information on how to implement the Classic theme in your project, refer to the following articles:
+
+* [Using the Classic Theme in Angular Projects](https://www.telerik.com/kendo-angular-ui/components/styling/theme-classic/)
```

Comment:
We don't have such pages on this URL yet.

Reply:
see nouvelle note